ci/pipeline.yml

```diff
@@ -290,13 +289,11 @@ resources:
     versioned_file: staging-prometheus-decrypted.yml
 
 - name: common-production
-  type: cg-common
+  type: s3-iam
```

Comment:
Now we can drop the `cg-common` resource type.

Reply:
Dropped.